cdk/app.ts

```diff
@@ -64,18 +64,14 @@ new PgStacInfra(app, buildStackName("pgSTAC"), {
     repoTag: stacBrowserRepoTag,
     customDomainName: stacBrowserCustomDomainName,
     certificateArn: stacBrowserCertificateArn,
+    webAclArn,
```

Comment:
this is something I was trying to figure out where to put it the first go around, the resource is technically just one per account at the moment - so anything in PgStacInfra that should go through a WAF would have access to it. I'm not sure what else needs to go through the WAF other than CF CDNs at the moment

Reply:
I moved it back to the top-level